dns/dnsconfig.js

```diff
@@ -34,6 +34,7 @@ D('testausserveri.fi', REG_NONE, DnsProvider(DNS_CLOUDFLARE),
     CNAME('openwilma', 'openwilma.github.io.'), // OpenWilma - https://github.com/OpenWilma/
     CNAME('tet', 'testausserveri.github.io.'), // TET - Niilas
     CNAME('datanomi', 'datanomi.net.', CF_PROXY_OFF), // datanomi - Cumpal
```

Comment:
Let's put these records in their own section.

```suggestion
    
    // Memes
    CNAME('datanomi', 'datanomi.net.', CF_PROXY_OFF), // datanomi - Cumpal
```